components/centraldashboard/frontend/index/index.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+    <title>Kubeflow</title>
+
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&amp;lang=en">
```

Comment:
People in China have problem accessing `googleapis.com` without VPN, so I think a better way is to use `parcel` or other thing to pre-pack the data, but that's a lot more work so I think it's sufficient to stay this way for now.